_docs/tasks/security/authn-policy.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Basic Istio authentication policy
+overview: This task shows you how to use Istio authentication policy to setup mutual TLS and simple end-user authentication.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+Through this task, you will learn how to:
```

Comment:
should we add a section to describe how to enable both?

Reply:
I add few sentences to the last section to mention peer authentication can be changed independently (from origin authn).